### ISSUE TITLE
Caused by: java.lang.NoClassDefFoundError: io/github/palexdev/virtualizedfx/flow/simple/SimpleVirtualFlow

### ISSUE THREAD

**Issue by ImEcuadorian** [palexdev/MaterialFX#256]
![image_2022-11-14_195149467](https://user-images.githubusercontent.com/46403054/201799484-e7b17370-6a9a-477f-b3eb-1cac5d029a50.png)

**Comment by palexdev** [comment]
In the previous issue you said you updated all your jars
I guess this may be the issue. As of now, MaterialFX is compatible only with VirtualizedFX versions <= 11.2.6

**Comment by ImEcuadorian** [comment]
But why if there is a higher version?

**Comment by ImEcuadorian** [comment]
> In the previous issue you said you updated all your jars
> I guess this may be the issue. As of now, MaterialFX is compatible only with VirtualizedFX versions <= 11.2.6

Yeah, that's the problem the newest version doesn't work correctly, so I will use 11.2.6 for VirtualizedFX thx so much
